src/battle/overlay_12_0224E4FC.c

```diff
@@ -6351,7 +6589,35 @@ u32 TryCriticalHit(BattleSystem *bsys, BATTLECONTEXT *ctx, int battlerIdAttacker
     return ret;
 }
 
-extern u16 sMetronomeUnuseableMoves[];
+static const u16 sMetronomeUnuseableMoves[] = {
+    MOVE_METRONOME, 
+    MOVE_STRUGGLE, 
+    MOVE_SKETCH, 
+    MOVE_MIMIC, 
+    MOVE_CHATTER, 
+    0xFFFE,
```

Comment:
possible to get a constant for this one?

Reply:
they're just table end markers, no real reason to use a const for them?

Reply:
I mean labelling them specifically as markers helps, I can see a possibility where they might get confused for proper data, though tbh whether that can actually happen is debatable